charts/enterprise-traces/values.yaml

```diff
@@ -20,33 +20,29 @@ externalLicenseVersion: "0"
 license:
   contents: "NOTAVALIDLICENSE"
 
-# TODO: Needs to fold into tempo-distributed
-storage:
+storage: &storage
   trace:
     backend: s3
     s3:
-      endpoint: |
-        {{ include "enterprise-traces.minio" . }}
-      bucket: enterprise-traces-blocks
+      # CHANGE lab-get TO YOUR NAMESPACE IN THE LINE BELOW
+      endpoint: lab-get-minio.lab-get.svc:9000
+      bucket: enterprise-traces-admin
```

Comment:
This is the data bucket. Lets name it `enterprise-traces-data`

Reply:
@basvdl looks good?